docs/windows/BUILD.md

```diff
@@ -0,0 +1,430 @@
+./build-wix6.sh [version]
+
+# Or use batch file directly (CMD/PowerShell)
+.\build-wix6.bat [version]
+```
+
+This uses WiX 4.x/6.x toolset (installed via `dotnet tool install`) to build the MSI installer.
+
+Output: `installer/bin/Release/PinShare-Setup.msi`
+
+#### On macOS/Linux
+
+WiX cannot run natively on macOS/Linux. Options:
+
+1. **CI/CD Pipeline:** Use GitHub Actions with Windows runners (recommended)
+2. **Windows VM:** Copy built binaries to Windows and run `build-wix6.bat`
+3. **Cross-compile binaries locally, build MSI in CI:** The `build-windows.sh` script creates all binaries; the MSI can be built by GitHub Actions
+
+#### Using CI/CD
+
+For automated builds, use GitHub Actions with Windows runners. See `.github/workflows/build.yml` for an example.
+
+## Troubleshooting Build Issues
+
+### CGO Errors
+
+**Error:** `gcc: command not found`
+
+**Linux/macOS Solution:**
+```bash
+# Install MinGW
+sudo apt-get install gcc-mingw-w64-x86-64  # Debian/Ubuntu
+brew install mingw-w64                      # macOS
+```
+
+**Windows Solution:**
+```
+Install TDM-GCC or MinGW-w64, add to PATH
+```
+
+**Error:** `undefined reference to...` (SQLite linking)
+
+**Solution 1:** Ensure CGO is enabled
+```bash
+export CGO_ENABLED=1
+export CC=x86_64-w64-mingw32-gcc  # Linux
+```
+
+**Solution 2:** Use pure-Go SQLite
+```bash
+# In go.mod, replace:
+# github.com/mattn/go-sqlite3
+# with:
+# modernc.org/sqlite
+
+# Then build without CGO
+CGO_ENABLED=0 go build ...
+```
+
+### IPFS Download Errors
+
+**Error:** `curl: (6) Could not resolve host`
+
+**Solution:** Check internet connection, try:
+```bash
+# Use wget instead
+wget https://dist.ipfs.tech/kubo/v0.31.0/kubo_v0.31.0_windows-amd64.zip
+```
+
+**Error:** `unzip: command not found`
+
+**Linux Solution:**
+```bash
+sudo apt-get install unzip
+```
+
+**Windows Solution:** Use PowerShell's `Expand-Archive` (see above)
+
+### WiX Errors
+
+**Error:** `candle.exe: command not found`
+
+**Solution:** Add WiX to PATH:
+```cmd
+set PATH=%PATH%;C:\Program Files (x86)\WiX Toolset v3.11\bin
+```
+
+**Error:** `The system cannot find the file specified`
+
+**Solution:** Ensure all binaries are built:
+```cmd
+dir ..\dist\windows\*.exe
+```
+
+All required files must exist before building the installer.
+
+## CI/CD Integration
+
+### GitHub Actions Example
+
+Create `.github/workflows/build-windows.yml`:
+
+```yaml
+name: Build Windows
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.24'
+
+    - name: Build Windows components
+      shell: bash
+      run: ./build-windows.bat
+
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v3
+      with:
+        name: pinshare-windows
+        path: |
+          dist/windows/*.exe
+          installer/bin/Release/*.msi
+```
```

Comment:
I don't think we need to provide an example so long as we have the actual workflow in the PR/repo.

```suggestion
```